server/src/main/java/io/unitycatalog/server/persist/dao/ColumnInfoDAO.java

```diff
@@ -41,12 +41,10 @@ public class ColumnInfoDAO {
   @Column(name = "name", nullable = false)
   private String name;
 
-  @Lob
-  @Column(name = "type_text", nullable = false, columnDefinition = "MEDIUMTEXT")
+  @Column(name = "type_text", nullable = false, length = 2048)
   private String typeText;
 
-  @Lob
-  @Column(name = "type_json", nullable = false, columnDefinition = "MEDIUMTEXT")
+  @Column(name = "type_json", nullable = false, length = 2048)
```

Comment:
This is significantly less than MEDIUMTEXT allows (16MB), and might not be enough to store some type json.

Reply:
Yes, I agree this is significantly less. I chose a number that seemed reasonable to hold a column type. I can increase the length limit to 16,777,215. This will cause hibernate to use type `MediumText` for MySQL backend and `text` for PostgreSQL.